Fluent/Physics.ipynb

```diff
@@ -6,70 +6,62 @@
+    "**In other words understanding physics and theory of problem before developing a CFD simulation can be thought of turning on a flash light before taking any step in a completely dark room. Hence, let's review the fundamental physics and theory behind the problem of \"2D Laminar Flow over a rotating cylinder\" as the first step for development and validation of CFD simulation for this problem of interest:**\n",
     "\n",
-    "Potential flow simplifies the problem by introducing a new variable called \"Streamfunction\" $\\psi$ which combines the two components of velocity into one (in a 2 dimensional problem). Now, instead of dealing with two variables, we have to deal with just one variable. We can assume the flow to be two-dimensional and incompressible and use the Laplce's Equation to find the Streamfunction. From the streamfunction we can find the velocity components. Bernoulli's Equation will give us the pressure distributions. This will give us a good estimate of the lift force.\n",
+    "To review the physics behind the flow over a rotating cylinder, the basic problem to be studied first is the flow over a stationary cylinder. Starting the review of a flow around a stationary cylinder will form the basis to leap into understanding and analyzing flow over a rotating cylinder. The problem of flow over a stationary cylinder was first studied using the \"Potential Flow Theory\", when the concept and understanding of \"Boundary Layer\" wasn't present yet (maybe cite the original paper). The fundamental assumption for the Potential Flow theory is neglecting the viscous effect within the flow field. This may not seem to be a great assumption at the beginning, since it is obvious that viscous effect play an important role in majority of fluid mechanics problems. However, this assumption helps to develop a better understanding of the generated resultant forces due to the flow and cylinder interaction. Furthermore, it can be shown that the viscous effect within the flow has minimal contribution into the lift force generation, which is the main variable of interest in the problem of flow over a rotating cylinder.\n",
     "\n",
-    "One more advantage of using Laplace's Equation is that it is linear. We can add streamfunctions of two flows to get the effect of the combined flow. A cylinder can be visualised by combinig an point source and a point sink close together in a uniform flow as shown in the figure 1 below. In cylindrical coordinates, the stream function can be writte as\n",
     "\n",
```

Comment:
Start with the assumption of neglecting viscous effects, then irrotational flow and then how $\psi$ can be developed and used. Make sure to build arguments from fundamentals See pp 103-109 of Lex Smith book.